jwst/associations/__init__.py

```diff
@@ -1,5 +1,4 @@
-
-__version__ = '0.9.20'
+__version__ = '0.9.22'
```

Comment:
We just completed release 0.10.0, so this should be bumped up accordingly (0.10.1?)

Reply:
👍 

About the documentation. If you are referring to [line 365](https://github.com/spacetelescope/jwst/pull/2307/files#diff-45c66a113fcc54865dc17ccf6190a464R365), where in the `finalize` statement, it refers to MSA specifically, yes, they are handled the same. I'll update the comment to be clearer. Otherwise, I'm not sure what you are referring to. There are two rules because the relevant keywords are different.